src/cow-react/modules/wallet/web3-react/hooks/useWalletMetadata.ts

```diff
@@ -25,6 +27,14 @@ export interface WalletMetaData {
   icon?: string
 }
 
+function getWcWalletIcon(meta: any) {
+  if (getIsAlphaWallet(meta.name)) {
```

Comment:
I guess this is a patch cause they don't send the image in their metadata?

Shouldn't we just let them know instead of doing these patches?